### ISSUE TITLE
Debug test feature runs all tests, not the chosen test

### ISSUE THREAD

**Issue by GorenDaniel** [vitest-dev/vscode#266]
**Describe the bug**
When right clicking on the test icon near the test lines, and choosing debug test, many debug workers are initiated and they are running the entire project, instead of the chosen test alone. It takes forever to reach my intended test.
Also, it reproduces in the vitest-ext-basic-example repo. Env setting are specified below.

Clicking on the debug test option:
![Screenshot 2024-02-24 at 18 37 52](https://github.com/vitest-dev/vscode/assets/104723536/4b6d18b8-298e-4d24-8fa6-7e212385ed45)

Multiple workers are running and the test keeps on spinning
![Screenshot 2024-02-24 at 18 38 15](https://github.com/vitest-dev/vscode/assets/104723536/e9a0a95e-20f4-46aa-b075-63399b59be29)

The debug console shows all test files
![Screenshot 2024-02-24 at 18 38 50](https://github.com/vitest-dev/vscode/assets/104723536/4383484b-7a58-4a62-b8d5-7c24417ac990)

**To Reproduce**

Have multiple test files, and click on debug test

**Expected behavior**
Only the specific test should run

**Environment**

package.json:
```json
{
    "name": "vitest-basic-example",
    "version": "1.0.0",
    "description": "",
    "main": "index.js",
    "scripts": {
        "test": "vitest --config vitest.config.ts"
    },
    "author": "",
    "license": "ISC",
    "devDependencies": {
        "envinfo": "^7.8.1",
        "jsdom": "^24.0.0",
        "vite": "^2.9.9",
        "vitest": "^1.3.0"
    }
}
```

vitest config:
```typescript
import os from "os";
import { defineConfig } from "vitest/config";

export default defineConfig({
    test: {
        include: ["**/*.test.ts"],
        environment: "jsdom",
        allowOnly: true,
        watch: true,
    },
});
```

Extension settings:
![Screenshot 2024-02-24 at 18 56 16](https://github.com/vitest-dev/vscode/assets/104723536/e4427e67-df0a-4e68-a1df-515f62a29470)

Test files:
- have multiple files.

- OS: macOS
- VSCode version: 1.85.2
- Vitest version:
- Vitest plugin version:

**Additional context**

**Comment by vangie** [comment]
<img width="1037" alt="image" src="https://github.com/vitest-dev/vscode/assets/1388582/1640d290-71c8-401d-a67c-31e90a5b398a">

**Comment by sheremet-va** [comment]
This should be fixed in pre-release 0.5.0 and higher. Note that the extension now requires Vitest 1.4.0 or higher.

Also, there is a known issue with debugger in the new version - sometimes it works only on a second click. You can follow the progress with that issue in #294
